lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js

```diff
@@ -373,7 +373,7 @@ define([
         /**
          * @param {Object} o
          */
-        openFileBrowser: function (o) {
+        openFileBrowser: _.debounce(function (o) {
```

Comment:
Hey, did you find out what is the reason that `openFileBrowser` is called multiple times in the first place?

Reply:
Hi @krzksz yes, this caused by this lines https://github.com/magento/magento2/blob/d56adbb6e91bb014feafa6febfde65a53b7d3cf3/lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js#L295 https://github.com/magento/magento2/blob/d56adbb6e91bb014feafa6febfde65a53b7d3cf3/lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js#L296

Where the same event fires two times with the same object, but as I observed this event needs both, but on some pages, this events triggered different